zarr/convenience.py

```diff
@@ -491,7 +492,7 @@ def __init__(self, log):
         elif callable(log):
             self.log_func = log
         elif isinstance(log, str):
-            self.log_file = io.open(log, mode="w")
+            self.log_file = _builtin_open(log, mode="w")
```

Comment:
Are we certain this is the only use of `io.open`?

Reply:
I think so. A grep of `io.open` yields:
```
$ fgrep -R io.open --exclude-dir=.git
zarr/convenience.py:            self.log_file = io.open(log, mode="w")
build/lib/zarr/convenience.py:            self.log_file = io.open(log, mode='w')
$ 
```
Do you have a specific concern, anything I may have overlooked?

Reply:
No not really. Just wanted to make sure we only have to do this once.